data-prepper-plugins/cwl-sink/src/main/java/org/opensearch/dataprepper/plugins/sink/config/CwlSinkConfig.java

```diff
@@ -0,0 +1,53 @@
+package org.opensearch.dataprepper.plugins.sink.config;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+
+public class CwlSinkConfig {
+    public static final String DEFAULT_BUFFER_TYPE = "in_memory";
+
+    //TODO: Change this to custom aws config class as its just a data container.
+    @JsonProperty("aws_config")
```

Comment:
We use "aws:" for this. Please follow the same convention.

Reply:
The name has been changed from 'aws_config' to 'aws'.